Makefile

```diff
@@ -52,10 +52,10 @@ include make/testing_flags.mk
 
 DEV_TAGS := $(if ${tags},$(DEV_TAGS) ${tags},$(DEV_TAGS))
 
-make_ldflags = $(shell echo -ldflags \"-X $(PKG)/build.Commit=$(COMMIT) \
```

Comment:
is it possible to move the `-ldflags` before `$(shell` to avoid duplication? so something like `-ldflags "$(shell ...)"`?

Reply:
I'm not really sure if the `make` syntax would allow for that, since the `make_ldflags` is interpreted as a callable procuedure/function. But I added it to the variables below. Still a bit of duplication but way less than before.

Reply:
i tested locally and it works :)